kubernetes/windows/main.ps1

```diff
@@ -311,6 +311,39 @@ function Set-EnvironmentVariables {
         Write-Host "Failed to set environment variable KUBERNETES_PORT_443_TCP_PORT for target 'machine' since it is either null or empty"
     }
 
+if (![string]::IsNullOrEmpty($isAADMSIAuth) -and $isAADMSIAuth.ToLower() -eq 'true') {
+    [System.Environment]::SetEnvironmentVariable("MONITORING_DATA_DIRECTORY", "C:\\opt\\genevamonitoringagent\\datadirectory", "Process")
```

Comment:
1. Move setting of all these env vars to function 2. have Process & Machine wide environment variables for envar one after other so that we can easily find if we missed set for both Process & Machine wide.